pkgs/development/tools/language-servers/vhdl-ls/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "vhdl-ls";
+  version = "0.64.0";
+
+  src = fetchFromGitHub {
+    owner = "VHDL-LS";
+    repo = "rust_hdl";
+    rev = "v${version}";
+    hash = "sha256-j5WRJJBUPKW3W+kY5hdqdZxxGkIAoEcW+A2pp23MX6Q=";
+  };
+
+  # No Cargo.lock upstream, see:
+  # https://github.com/VHDL-LS/rust_hdl/issues/166
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
+  postPatch = ''
+    ln -s ${./Cargo.lock} Cargo.lock
+  ''
+  # Also make it look up vhdl_libraries in an expected location
+  + ''
+    substituteInPlace vhdl_lang/src/config.rs \
+      --replace /usr/lib $out/lib
+  '';
+
+  postInstall = ''
+    mkdir -p $out/lib/rust_hdl
+    cp -r vhdl_libraries $out/lib/rust_hdl
+  '';
+
+  meta = with lib; {
+    description = "a fast VHDL language server";
+    homepage = "https://github.com/VHDL-LS/rust_hdl";
+    license = licenses.mpl20;
+    mainProgram = "vhdl_ls";
+    maintainers = with maintainers; [ doronbehar ];
```

```suggestion
    maintainers = [ lib.maintainers.doronbehar ];
```
Reasoning for avoiding `with`:
  - https://github.com/atorres1985-contrib/nixpkgs/commit/451dfa6d84586b5cb83e6a7254689dff66b4523a